app-modules/ai/src/Settings/AiSettings.php

```diff
@@ -53,6 +53,10 @@ public static function group(): string
         return 'ai';
     }
 
+    /**
+     *
```

Comment:
Please fix this formatting, the empty line is not required. Please check other places